pkgs/applications/misc/lifelines/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, autoconf, libtool, automake, pkgconfig
+, gettext, libiconv, bison, ncurses, perl, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  version = "2017-06-12";
+  name = "lifelines-unstable-${version}";
+
+  src = fetchFromGitHub {
+    owner = "MarcNo";
+    repo = "lifelines";
+    rev = "61974641d9e2ca6ee808597d934b0f70eab69a56";
+    sha256 = "1nqn395pljwdz3rbanqcakzs00svqk53xjazfp66rqzia7wdh0s5";
+  };
+
+  buildInputs = [
+    autoconf
```

Comment:
`auto*` and `libtool` at least will be pulled in by `autoreconfHook`

`pkgconfig` and `bison` could probably be moved to `nativeBuildInputs` (as a rule of thumb, tools that only run during the build can go in native).